tests/testit/test-base64.R

```diff
@@ -31,3 +31,23 @@ assert('base64_encode_r() returns the same result as base64_encode()', {
   f = R_logo()
   base64_encode_r(f) %==% base64_encode(f)
 })
+
+assert('base64_decode() does not accept a non-string input', {
+  has_error(base64_decode(x = 42))
+})
+
+assert('base64_decode() does not accept both string and file input', {
+  f = tempfile()
+  has_error(base64_decode(x = 'Kg==', from = f))
+})
+
+assert('base64_decode() returns the same result when the same string is used as an input directly or from a file connection', {
+  f = tempfile()
+  writeLines(text = "Kg==", con = f, sep = "")
+  base64_decode(from = f) %==% base64_decode(x = 'Kg==')
+})
+
+assert('base64_uri() returns proper data type', {
+  f = R_logo()
+  unlist(strsplit(base64_uri(f), split = ';'))[1] %==% 'data:image/svg+xml'
```

Comment:
Let's not assume `R_logo()` always returns the svg image. You can look at the source code of `R_logo()`.

You can add a condition like this:

```suggestion
 if (grepl('[.]svg$', f)) strsplit(base64_uri(f), split = ';')[[1]] %==% 'data:image/svg+xml'
```